datafusion/ffi/src/udaf/accumulator.rs

```diff
@@ -70,6 +70,10 @@ pub struct FFI_Accumulator {
     /// Internal data. This is only to be accessed by the provider of the accumulator.
     /// A [`ForeignAccumulator`] should never attempt to access this data.
     pub private_data: *mut c_void,
+
+    /// Utility to identify when FFI objects are accessed locally through
```

Comment:
I feel like this is a super important new api -- maybe can you add a doc link to somewhere that descsribes the high level `library_marker_id` concept and 
1. how it should be used?
2. how users should choose a library marker (how do we ensure no conflicts?)

Reply:
Perhaps link to `pub extern "C" fn get_library_marker_id() -> usize {`